tests/pytests/test_kaguya_drivers.py

```diff
@@ -1,119 +1,185 @@
-        mock_target_name.return_value = 'MOOn'
-        assert driver.reference_frame == 'MOON_ME'
-        mock_target_name.return_value = 'sun'
-        assert driver.reference_frame == 'NO TARGET'
+    assert driver.reference_frame == 'MOON_ME'
 
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
 def test_focal2pixel_samples(driver):
-    assert driver.focal2pixel_samples == [0, 0, -1]
+    with patch('ale.drivers.selene_drivers.spice.gdpool', return_value=[2]) as gdpool, \
+         patch('ale.drivers.selene_drivers.spice.bods2c', return_value=-12345) as bods2c:
+        assert driver.focal2pixel_samples == [0, 0, -1/2]
+        gdpool.assert_called_with('INS-12345_PIXEL_SIZE', 0, 1)
 
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
 def test_focal2pixel_lines(driver):
-    assert driver.focal2pixel_lines == [0, -1, 0]
-
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
-def test_odkx(driver):
-    assert driver._odkx == [1, 1, 1, 1]
-
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
-def test_odky(driver):
-    assert driver._odky == [1, 1, 1, 1]
-
-def test_line_exposure_duration(driver):
-    with patch.dict(driver.label, {'CORRECTED_SAMPLING_INTERVAL':
-        pvl._collections.Units(value=1000, units='<msec>')}) as f:
-        assert driver.line_exposure_duration == 1
-    with patch.dict(driver.label, {'CORRECTED_SAMPLING_INTERVAL':
-        [pvl._collections.Units(value=10000, units='<msec>'), 1]}) as f:
-         assert driver.line_exposure_duration == 10
-
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
-def test_focal_length(driver):
-    assert driver.focal_length == 1
-
-@patch('ale.base.label_pds3.Pds3Label.instrument_id', 123)
-def test_usgscsm_distortion_model(driver):
-    distortion_model = driver.usgscsm_distortion_model
-    assert distortion_model['kaguyatc']['x'] == [1, 1, 1, 1]
-    assert distortion_model['kaguyatc']['y'] == [1, 1, 1, 1]
-
-def test_detector_start_sample(driver):
-    with patch.dict(driver.label, {'FIRST_PIXEL_NUMBER': 123}) as f:
-        assert driver.detector_start_sample == 123
-
-def test_sensor_model_version(driver):
-    assert driver.sensor_model_version == 1
+    with patch('ale.drivers.selene_drivers.spice.gdpool', return_value=[2]) as gdpool, \
+         patch('ale.drivers.selene_drivers.spice.bods2c', return_value=-12345) as bods2c:
+        assert driver.focal2pixel_lines == [0, -1/2, 0]
+        gdpool.assert_called_with('INS-12345_PIXEL_SIZE', 0, 1)
+
+def test_no_metakernels(driver, tmpdir, monkeypatch):
+    monkeypatch.setenv('ALESPICEROOT', str(tmpdir))
+    reload(ale)
+
+    with pytest.raises(ValueError):
+        with driver as failure:
+            pass
+
+def test_no_spice_root(driver, monkeypatch):
+    monkeypatch.delenv('ALESPICEROOT', raising=False)
+    reload(ale)
+
+    with pytest.raises(EnvironmentError):
+        with driver as failure:
+            pass
```

Comment:
I think that the no metakernels and no spice root is being tested in test_env.py

Reply:
Yeah, there's no Selene specific code here, so this shouldn't be tested.

Reply:
So can these two tests be removed?

Reply:
yes